server/layout.js

```diff
@@ -40,6 +40,11 @@ module.exports = function(state, body = '') {
           type="text/css"
           href="${assets.get('app.css')}"
         />
+        <link
```

Comment:
An if-statement should be used here, to only include this if a custom stylesheet is set (not `null` or empty string).

Reply:
Added a check to see if the user is using custom CSS using a JS script in the head tag.

In effect the default value assigned to UI_CUSTOM_CSS is 'undefined'. 

The script will only create the html link if state.js returns the value for UI_CUSTOM_CSS as anything but undefined.

Tested it in the following scenarios:

* Having hashed out the environment variable completely.
* Having added the environment variable but with no value.
* Having added the environment variable but with the value of '' (empty string).

In all three cases the value of 'undefined' was returned correctly.